data/json/recipes/chem/fuel.json

```diff
@@ -0,0 +1,108 @@
+[
+  {
+    "result": "chem_ethanol",
+    "type" : "recipe",
+    "byproducts": [ "water_clean" ],
+    "category": "CC_CHEM",
+    "subcategory": "CSC_CHEM_FUEL",
+    "skill_used": "cooking",
+    "difficulty": 5,
+    "time": 30000,
+    "book_learn": [ [ "textbook_chemistry", 5 ], [ "brewing_cookbook", 4 ] ],
+    "qualities": [
+        { "id": "BOIL", "level": 2, "amount": 1 },
+        { "id": "BREW", "level": 2, "amount": 1 }
+     ],
+     "tools": [
+       [ [ "fire", -1 ], [ "hotplate", 30 ], [ "toolset", 30 ] ],
+       [ [ "thermometer", -1 ] ]
+     ],
+     "components": [ [
+       [ "vodka", 14 ],
+       [ "whiskey", 14 ],
+       [ "single_malt_whiskey", 14 ],
+       [ "brandy", 14 ],
+       [ "rum", 14 ],
+       [ "gin", 14 ],
+       [ "tequila", 14 ],
+       [ "bum_wine", 56 ],
+       [ "fruit_wine", 56 ],
+       [ "moonshine", 14 ],
+       [ "mixed_alcohol_strong", 14 ]
+    ] ]
+  },
+  {
+    "result": "denat_alcohol",
+    "type" : "recipe",
+    "byproducts": [ "water_clean" ],
+    "category": "CC_CHEM",
+    "subcategory": "CSC_CHEM_FUEL",
+    "skill_used": "cooking",
+    "difficulty": 3,
+    "time": 30000,
+    "book_learn": [ [ "textbook_chemistry", 4 ], [ "brewing_cookbook", 3 ] ],
+    "qualities": [
+        { "id": "BOIL", "level": 1, "amount": 1 },
+        { "id": "BREW", "level": 1, "amount": 1 }
```

Comment:
Yes, it's somewhat a simplification. I want to represent poor quality `dirty_ethanol and industrial grade `clean_ethanol` so I think that requiring a chemistry kit vs proper still and thermometer is enough detail.

The whole of our brewing/distilling recipes are a bit messed up. c8b11e9 provides some sanity in that `BREW` becomes `DISTILL` (you cannot brew wine using a still) and the sugar refining recipe (which is also not brewing) is separated out to have requirements similar to the IRL process. I'm not going to tackle the brewing recipes as someone else has already started work on fermentation vats and that would be the best route so for now those recipes have been changed to require `COOK`.